vector/src/main/java/im/vector/app/features/location/live/map/LocationLiveMapViewFragment.kt

```diff
@@ -50,7 +53,8 @@ import javax.inject.Inject
  */
 class LocationLiveMapViewFragment @Inject constructor(
         private var urlMapProvider: UrlMapProvider,
-) : VectorBaseFragment<FragmentSimpleContainerBinding>() {
+        private var bottomSheetController: LiveLocationBottomSheetController,
+) : VectorBaseFragment<FragmentLocationLiveMapViewBinding>() {
```

Comment:
I think you can remove the `fragment_simple_container.xml`.

Reply:
Right, removed.